README.md

```diff
@@ -112,8 +112,8 @@ Note: Upstart/SysV init based OS types are not supported.
 ## Supported Components
 
 - Core
-  - [kubernetes](https://github.com/kubernetes/kubernetes) v1.16.7
-  - [etcd](https://github.com/coreos/etcd) v3.3.10
+  - [kubernetes](https://github.com/kubernetes/kubernetes) v1.16.8
+  - [etcd](https://github.com/coreos/etcd) v3.3.12
```

Comment:
Is this PR updating etcd version or was the readme out of date?

Reply:
I cherry-picked 0330442c635478f2c51002368e876cd79b17f66a which doesn't contain etcd change. But I resolved some conflicts and it did squeeze in while doing it. Do you want to include etcd 3.2.12 in that PR or you prefer to stick to 3.3.10?

Reply:
We could backport both.